packages/db/src/types/granule.ts

```diff
@@ -23,6 +23,8 @@ export interface PostgresGranule {
   // Processing info from execution
   processing_start_date_time?: Date,
   processing_end_date_time?: Date,
+  data_type?: string,
+  version?: string,
```

Comment:
Is it worthy to mention these changes and the ones above in the changelog? I noticed we didn't do that for https://github.com/nasa/cumulus/pull/2026/files.